test/unit/redis_connection_error_test.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+require 'test_helper'
+
+class RedisConnectionErrorTest < ActiveSupport::TestCase
+  test 'Name or service not known' do
+    redis_config = ThreeScale::RedisConfig.new(
+      url: 'redis://mymaster',
+      sentinels: 'invalid_host:26380,other_invalid_host:26381',
+      role: :master
+    ).config
+
+    redis = Redis.new(redis_config)
+
+    exception = assert_raises(RedisClient::CannotConnectError) { redis._client.send(:connect) }
+    assert_equal 'No sentinels available', exception.message
+  end
```

Comment:
what is this testing?

Reply:
It's the same as [test/unit/redis_hacks_test.rb](https://github.com/3scale/porta/pull/3576/files/23989575f8ed1bd50b485c401d68e232927add34#diff-08a52b7fd78ba4a94f833530f263af035a22a39ab5feed4d35cd1849e7cd0abe). Just I renamed it to still test the behavior, even when we removed the monkey patch.